### PR TITLE
Added public initializer for KeyParams.

### PR DESCRIPTION
**Pull request by mannuch**
# Pull Request

## Related issue
The `KeyParams` and `KeysQuery` structs do not have publicly accessible initializers. This implies that any code outside the `MeiliSearch` module cannot create new instances of these structs.

This works in direct contrast to examples laid out in the Meilisearch documentation such as when it displays constructing new `KeyParams` instances to pass to the `POST /keys` endpoint, which can be seen [here](https://docs.meilisearch.com/learn/security/master_api_keys.html#creating-an-api-key), if you select Swift as the client language.

I've simply explicitly added a public initializers to both the `KeyParams` and `KeysQuery` structs.

### CODE REVIEW

**Comment by curquiza** [comment]
bors try

**Comment by curquiza** [comment]
Hello @mannuch 
Thanks for the PR!
Looks like there is a linter issue. Could you fix it?

**Comment by brunoocasali** [comment]
Hi @mannuch thanks for your contribution!!

I would like to know if you have any tips to test this behavior, so we can prevent that from happening again in the future.

**Review by brunoocasali — CHANGES_REQUESTED** [review]
Thanks a lot @mannuch! As @curquiza said, we still have some linting issues. Also, I posted a comment earlier asking about how we can test this :)

**Comment by brunoocasali** [comment]
Hi, @mannuch are you still interested in continuing working in this PR?

**Comment by mannuch** [comment]
Yes I am! Sorry I've been busy the past couple weeks, but I will try to implement the necessary changes at the end of the week.

Thanks for your patience!

**Review by brunoocasali — CHANGES_REQUESTED** [review]

File: Sources/MeiliSearch/Model/KeyParams.swift [hunk 1]
```diff
@@ -19,6 +19,14 @@ public struct KeyParams: Codable, Equatable {
   public let indexes: [String]
   /// Date and time when the key will expire, represented in RFC 3339 format.
   public let expiresAt: String?
+  public init(description: String? = nil, name: String? = nil, uid: String? = nil, actions: [String], indexes: [String], expiresAt: String?) {
```
Comment (brunoocasali) [inline Sources/MeiliSearch/Model/KeyParams.swift:22]:
```suggestion

  public init(description: String? = nil, name: String? = nil, uid: String? = nil, actions: [String], indexes: [String], expiresAt: String?) {
```

File: Sources/MeiliSearch/Model/KeyParams.swift [hunk 2]
```diff
@@ -20,6 +20,15 @@ public struct KeyParams: Codable, Equatable {
...
 
+  public init(description: String? = nil, name: String? = nil, uid: String? = nil, actions: [String], indexes: [String], expiresAt: String?) {
+      self.description = description
+      self.name = name
+      self.uid = uid
+      self.actions = actions
+      self.indexes = indexes
+      self.expiresAt = expiresAt
+  }
+
```
Comment (brunoocasali) [inline Sources/MeiliSearch/Model/KeyParams.swift:31]:
```suggestion
  public init(description: String? = nil, name: String? = nil, uid: String? = nil, actions: [String], indexes: [String], expiresAt: String?) {
    self.description = description
    self.name = name
    self.uid = uid
    self.actions = actions
    self.indexes = indexes
    self.expiresAt = expiresAt
  }

```

**Review by brunoocasali — APPROVED** [review]
Thanks for contributing with Meilisearch ❤️ 

bors merge!

**Comment by brunoocasali** [comment]
Thanks again for contributing to Meilisearch :heart:
If you are participating in [Hacktoberfest](https://blog.meilisearch.com/contribute-hacktoberfest-2022/), and you would like to receive some gift from Meilisearch too, please complete [this form](https://meilisearch.typeform.com/to/idt5qugY).

**Pull request merged**